templates/security/esm.html

```diff
@@ -8,318 +8,555 @@
+          </tr>
+          <tr class="u-hide--small u-hide--medium" style="border-top: 0;">
+            <td width="50%">&nbsp;</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">GA</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">1 year</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">2 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">3 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">4 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">5 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">6 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">7 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">8 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">9 years</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">10 years</td>
+          </tr>
+          <tr class="u-hide--large" style="border-top: 0;">
+            <td width="50%">&nbsp;</td>
+            <td class="u-align--right" style="border-right: 1px solid #E5E5E5;">GA</td>
+            <td colspan="5" class="u-align--right" style="border-right: 1px solid #E5E5E5;">5 years</td>
+            <td colspan="5" class="u-align--right" style="border-right: 1px solid #E5E5E5;">10 years</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </section>
+
+  <section class="p-strip is-deep u-no-padding--top">
     <div class="row">
-      <div class="col-8">
-        <h2>Is Ubuntu 16.04 LTS still supported beyond April 2021?</h2>
+      <h3 class="p-text--x-small-capitalised">Case studies</h3>
+      <hr />
+      <div class="col-3">
+        <div style="max-width: 118px;">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/e32cef6a-aa62411e-8a50-467b-9b76-edb083819144.svg",
+            alt="",
+            width="1224",
+            height="792",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+      <div class="col-9">
+        <h4>
+          Interana uses ESM while planning public cloud upgrades to 18.04
+        </h4>
         <p>
-          Ubuntu 16.04 LTS will still be supported beyond its free initial five-year maintenance period in April 2021, as it transitions to the extended security maintenance phase - with five additional years of security ensured.
+          Due to the large amounts of customer data that Interana handles, ensuring that security was tight was a priority. Rather than rush the upgrade from Ubuntu 14.04LTS, which would have been a major inconvenience for its customers, Interana turned to Canonical and ESM.
```

Comment:
```suggestion
          Due to the large amounts of customer data that Interana handles, ensuring that security was tight was a priority. Rather than rush the upgrade from Ubuntu 14.04 LTS, which would have been a major inconvenience for its customers, Interana turned to Canonical and ESM.
```